src/dsm/headers/DijkstraWeights.hpp

```diff
@@ -0,0 +1,18 @@
+
+#ifndef dijkstra_weights_hpp
```

Comment:
Shall we start using `#pragma once`?

Reply:
We should do a PR where we convert all of them